plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -0,0 +1,318 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
```

Comment:
At the beginning my intention was to make DoubleKeyValue a generic widget. Now it's difficult to separate widget from API.